### PR TITLE
Solves exercise ex8

### PR DESCRIPTION
**Pull request by PedroPapa1**
8. **Sort Two Values**  
   Write an algorithm to read 2 values (consider that equal values will not be read) and write them in ascending order.

### CODE REVIEW

File: ex8/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    if ( valueOne === valueTwo) {
+        return ("The values can't be the same.");
+    }
+
+    if(valueOne < valueTwo) {
+        console.log(`The values in ascending order: ${valueOne}, ${valueTwo}`);
+    } else {
+        console.log(`The values in ascending order: ${valueTwo}, ${valueOne}`);
+    }
+}
```
Comment (emiliosheinz) [inline ex8/index.js]: You are mixing console.log and return statements in the same function. Please, keep it consistent.

File: ex8/index.js
Comment (emiliosheinz) [inline ex8/index.js:1]: Even though this works well for two numbers, I would like you to dig a bit deeper and think about a scalable solution that can easily be adapted for more numbers.

File: ex8/index.js (hunk 1 above)
Comment (PedroPapa1) [reply ex8/index.js]: In this case, I had thought about using return to "break" the function if the values ​​are the same, so is it better to put them all in return?

File: ex8/index.js
Comment (PedroPapa1) [reply ex8/index.js:1]: Cool, I'll think of a better way to do this.

File: ex8/index.js (hunk 1 above)
Comment (emiliosheinz) [reply ex8/index.js]: The thing is, a function should consistently return something or not return anything (void). In this case, the return type is both string and void which is not something valid. Therefore I'd suggest always simply logging the string or instead, always returning it.

**Review by VictorComette — APPROVED** [review]

**Pull request merged**